modules/adagioAnalyticsAdapter.js

```diff
@@ -340,10 +370,14 @@ let adagioAdapter = Object.assign(adapter({ emptyUrl, analyticsType }), {
         case CONSTANTS.EVENTS.BID_RESPONSE:
           handlerBidResponse(args);
           break;
+        case CONSTANTS.EVENTS.AUCTION_END:
+          handlerAuctionEnd(args);
+          break;
         case CONSTANTS.EVENTS.BID_WON:
           handlerBidWon(args);
           break;
-        case CONSTANTS.EVENTS.AD_RENDER_SUCCEEDED:
+        // AD_RENDER_SUCCEEDED seems redundant with BID_WON.
+        // case CONSTANTS.EVENTS.AD_RENDER_SUCCEEDED:
         case CONSTANTS.EVENTS.AD_RENDER_FAILED:
           handlerAdRender(args, eventType === CONSTANTS.EVENTS.AD_RENDER_SUCCEEDED);
```

Comment:
The `eventType` param becomes useless?

Reply:
As long as we keep it commented, wouldn't we prefer to keep the eventType logic just in case we uncomment it ?
Or we drop it, rename `handlerAdRender(args, bool)` -> `handlerAdRenderFailed(args)`

Reply:
Yeah you right, let's keep your actual code